File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex14.java
//Importando Scanner
import java.util.Scanner;

public class Ex14{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	//vars
	int anoNasc, anoAtual, anos, meses, dias, semanas;
	
	System.out.println("Idade em anos, meses, dias e semanas\n\n");
	
	//Entrada	
	System.out.print("Insira o ano de nascimento: ");
	anoNasc = input.nextInt();
	
	System.out.print("Insira o ano atual: ");
	anoAtual = input.nextInt();
	
	//Processamento
	anos = anoAtual - anoNasc;
	meses = anos * 12;
	semanas = meses * 4;
	dias = semanas * 7;
	
	//Saída
	System.out.println("Idade: ");
	System.out.println("\t-anos: " + anos);
	System.out.println("\t-meses: " + meses);
	System.out.println("\t-semanas: " + semanas);
	System.out.println("\t-dias: " + dias );
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex14.c
#include <stdio.h>

main()
{
    //vars
    int anoNascimento, anoAtual, anos, meses, dias, semanas;

    printf("\t\t\tIdade em anos, meses, dias e semanas\n\n");

    //Entrada
    printf("Insira seu ano de nascimento: ");
    scanf("%d",&anoNascimento);

    printf("Insira o ano atual: ");
    scanf("%d",&anoAtual);

    //Processamento
    anos = anoAtual - anoNascimento;
    meses = anos*12;
    semanas = meses*4;
    dias = semanas*7;

    //Saída
    printf("Idade aproximada em: \n");
    printf("-anos: %d\n", anos);
    printf("-meses: %i\n", meses);
    printf("-semanas: %i\n", semanas);
    printf("-dias: %in", dias);




}
